1. letters가 바인딩하는 문자열에서 첫번째와 세번째 문자를 출력하세요.  
`letter = 'python'`

In [1]:
letter = "python"
print(letter[0], letter[2])

p t


2. 자동차 번호가 다음과 같을 때 뒤에 4자리만 출력하세요.  
`license_plate = "24가 2210"` 실행 예: `2210`

In [2]:
license_plate = "24가 2210"
print(license_plate[4:])

2210


In [3]:
# 답은
print(license_plate[-4:])

2210


3. 아래의 문자열에서 '홀' 만 출력하세요.  
`string = "홀짝홀짝홀짝"`

In [4]:
string = "홀짝홀짝홀짝"
print(string[::2]) # [시작인덱스:끝인덱스:오프셋]

홀홀홀


4. 문자열을 거꾸로 뒤집어 출력하세요.  
`string = "PYTHON"`

In [5]:
string = "PYTHON"
print(string[::-1])

NOHTYP


5. 아래의 전화번호에서 하이푼 ('-')을 제거하고 출력하세요.  
`phone_number = "010-1111-2222"`

In [6]:
phone_number = "010-1111-2222"
print(phone_number.replace("-", ""))

01011112222


In [7]:
# 답은 중간에 띄어쓰기가 있었어야 했음
print(phone_number.replace("-", " "))

010 1111 2222


6. 25번 문제의 전화번호를 아래와 같이 모두 붙여 출력하세요.  
실행 예: `01011112222`

In [8]:
# 위에

7. url 에 저장된 웹 페이지 주소에서 도메인을 출력하세요.  
`url = "http://sharebook.kr"` 실행 예: `kr`

In [9]:
url = "http://sharebook.kr"
print(url[-2:])

kr


In [10]:
# 답은
url_split = url.split('.') # .을 기준으로 분리
print(url_split[-1]) # 마지막만 인덱싱

kr


8. 아래 코드의 실행 결과를 예상해보세요.
```
lang = 'python'
lang[0] = 'P'
print(lang)
```

In [12]:
# 'Python'이겠지
lang = 'python'
lang[0] = 'P'
print(lang)

TypeError: ignored

In [13]:
# 문자열은 수정할 수 없다...

9.아래 문자열에서 소문자 'a'를 대문자 'A'로 변경하세요.  
`string = 'abcdfe2a354a32a'`

In [14]:
string = 'abcdfe2a354a32a'
print(string.replace("a", "A"))

Abcdfe2A354A32A


10. 아래 코드의 실행 결과를 예상해보세요.
```
string = 'abcd'
string.replace('b', 'B')
print(string)
```

In [15]:
# 'aBcd'일것임
string = 'abcd'
string.replace('b', 'B')
print(string)

abcd


In [16]:
# `abcd`가 그대로 출력됩니다.
# 왜냐하면 문자열은 변경할 수 없는 자료형이기 때문입니다.
# replace 메서드를 사용하면 원본은 그대로 둔채로 변경된 새로운 문자열 객체를 리턴해줍니다.